x-pack/plugins/endpoint/public/applications/endpoint/view/managing/url_from_query_params.ts

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
I'm probably going to need something like this as well - wondering if we should make it generic and add it to `endpoint/lib/`